WebisD/http-api-without-lib
# Fix bugs in multiple requests and add colored buttons for different responses

### **Why**
- When clicking on submit buttons multiple times on end the server would not register the entries with proper ids. There's also the need for visual confirmation on whether the requests were successful or not.

### **What was done**
- Created function HandlerDatabase::arePokemonsEqual() that returns whether two given dictionaries representing pokemons have the same data.
- Changed buttons colors on XMLHttpRequest and set a timeout to the original colors.

### **How to test**
- Run the `main.py` file.
- Open your browser and go to `localhost:8080/post`
- Click 'Add" without filling any data. The result should be the same as below: <br>
![Hnet com-image (1)](https://user-images.githubusercontent.com/63318342/116016740-fb8f6400-a613-11eb-9af2-1f88295d73be.gif)

- Click 'Add' after filling the data correctly. The result should be the same as below: <br>
![Hnet com-image](https://user-images.githubusercontent.com/63318342/116016759-03e79f00-a614-11eb-9f84-f250d8d3455a.gif)
